### PR TITLE
[Service] Check return value

### PR DESCRIPTION
**Pull request by gichan-jang**
- Fix unchecked return value.

### CODE REVIEW

File: c/src/ml-api-remote-service.c [hunk 1]
```diff
@@ -218,13 +218,27 @@ _mlrs_model_register (gchar * service_key, nns_edge_data_h data_h,
...
+    goto error;
+  }
+
   active_bool = _mlrs_parse_activate (activate);
 
   dir_path = g_build_path ("/", current_dir, service_key, NULL);
-  g_mkdir_with_parents (dir_path, 0755);
+  if (g_mkdir_with_parents (dir_path, 0755) < 0) {
+    _ml_loge ("Failed to create directory %s., error: %s", dir_path,
+        error ? error->message : "unknown error");
```
Comment (jaeyun-jung) [inline c/src/ml-api-remote-service.c:236]: invalid param error (it is not set)

Comment (gichan-jang) [reply c/src/ml-api-remote-service.c]: Oh, Thanks. Updated.

File: c/src/ml-api-remote-service.c [hunk 2]
```diff
@@ -562,28 +584,55 @@ ml_remote_service_register (ml_service_h handle, ml_option_h option, void *data,
...
+  }
+  ret = nns_edge_data_set_info (data_h, "service-key", service_key);
+  if (NNS_EDGE_ERROR_NONE != ret) {
+    _ml_error_report ("Failed to set service key in edge data.");
+    goto done;
+  }
+  ret = ml_option_get (option, "description", (void **) &description);
+  if (ML_ERROR_NONE != ret) {
+    _ml_logi ("Failed to get option description.");
+  }
```
Comment (songgot) [inline c/src/ml-api-remote-service.c:600]: Does it need ```goto done``` ?

File: c/src/ml-api-remote-service.c [hunk 3]
```diff
@@ -218,13 +218,26 @@ _mlrs_model_register (gchar * service_key, nns_edge_data_h data_h,
...
+    _ml_loge ("Failed to get info from data handle.");
+    ret = FALSE;
+    goto error;
+  }
+
   active_bool = _mlrs_parse_activate (activate);
 
   dir_path = g_build_path ("/", current_dir, service_key, NULL);
-  g_mkdir_with_parents (dir_path, 0755);
+  if (g_mkdir_with_parents (dir_path, 0755) < 0) {
```
Comment (wooksong) [inline c/src/ml-api-remote-service.c:234]: Before making a new directory, is it necessary to check that the target directory exists?

Comment (gichan-jang) [reply c/src/ml-api-remote-service.c:234]: If the directory already exists, this returns 0. So I think don't need to check the the existence of the directory.

File: c/src/ml-api-remote-service.c (hunk 2 above)
Comment (gichan-jang) [reply c/src/ml-api-remote-service.c:600]: It's not necessary option, so just inform to the user.

**Review by wooksong — APPROVED** [review]
LGTM

**Review by myungjoo — APPROVED** [review]

**Pull request merged**